load.py

```diff
@@ -34,15 +34,26 @@
     if os.path.exists(x) and os.path.exists(y):
         full_dataset.append((x, y))
 
+print 'full dataset size : %i' % len(full_dataset)
```

Comment:
%d is the way to display integers in format strings

https://docs.python.org/3.4/library/string.html